src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -353,17 +353,48 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_U16(shape.rot.z, 0, ICHAIN_STOP),
 };
 
-static AnimationHeader* sAnimations[] = {
-    &gameplay_keep_Anim_020658, &gameplay_keep_Anim_022CA8, &gameplay_keep_Anim_020658, &gameplay_keep_Anim_022E68,
-    &gameplay_keep_Anim_0204B4, &gameplay_keep_Anim_022BE8, &gameplay_keep_Anim_022D90, &gameplay_keep_Anim_022BE8,
-    &gameplay_keep_Anim_022FF0, &gameplay_keep_Anim_0205A0,
-};
-static u8 sAnimOpenFrames[10] = {
-    25, 25, 25, 25, 25, 25, 25, 25, 25, 25,
-};
-
-static u8 sAnimCloseFrames[10] = {
-    60, 60, 60, 70, 70, 60, 60, 60, 60, 70,
+static AnimationHeader* sAnimations[2 * PLAYER_FORM_MAX] = {
+    // door A
```

Comment:
Based on the player code, these are separated by door direction. Perhaps use that in the comments instead of just `A` and `B`?

Reply:
They are yeah. I knew it was left/right, but I wasn't sure which was A/B (taken from OG names) but I worked it out and updated it